runtime/examples/run.cpp

```diff
@@ -16,27 +16,34 @@ limitations under the License.
 #include <string>
 #include <iostream>
 #include <vector>
+#include <cstring>
 
 #include "blueoil.hpp"
 
-// TODO: delete this func. it is for debug.
-blueoil::Tensor RandomImage(int height, int width, int channel) {
-    blueoil::Tensor t({height, width, channel});
-    std::vector<float>& data = t.data();
-    unsigned seed = 0;
 
-    for (size_t i = 0; i < data.size(); ++i) {
-        const float f_rand = static_cast <float> (rand_r(&seed)) / static_cast <float> (RAND_MAX) * 255;
-        data[i] = f_rand;
+int main(int argc, char **argv) {
+  std::string imagefile, meta_yaml;
+
+  for (int i = 1 ; i < (argc-1); i++) {
```

Comment:
Could you delete the space to the left of the `;`?

Also, could you make the same modification for all for loops below?

Reply:
https://github.com/blue-oil/blueoil/pull/577/commits/ac1e423c1bc34800803e1a63ae81e47d1c851fd4